modules/installation-bootstrap-gather.adoc

```diff
@@ -34,9 +34,9 @@ the bootstrap and control plane machines:
 ** If you used installer-provisioned infrastructure, run the following command:
 +
 ----
-$ ./openshift-install gather bootstrap --dir=<directory> <1>
+$ ./openshift-install gather bootstrap --dir=<installation_directory> <1>
```

Comment:
Someone replaced <installation_directory> with <directory> but didn't update the callouts below. It could go either way, but I'm guessing someone made this change unintentionally when they were replacing text across a number of different files. In this case, "installation directory" is the more familiar term because the customer just saw it in the installation instructions.